themes/luci-theme-material/htdocs/luci-static/material/cascade.css

```diff
@@ -1403,9 +1405,21 @@ body:not(.Interfaces) .cbi-rowstyle-2:first-child {
 	background-color: var(--red-color-high);
 }
 
+/* indication line for drag-and-drop in UIDynamicList*/
+.cbi-dynlist > .item.drag-over {
+	border-top: 1px solid black;
+}
+
+/* transluce item being dragged in UIDynamicList*/
```

Comment:
translucent?

Reply:
Category any noun can be verbed :) 
I was trying to save some bytes, ill make it "make item partially transparent while being dragged in UIDynamicList"